proto/authorization/authorization.proto

```
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package aegis.authorization.v1;
```

Buf (1.65.0)

[error] Files with package "aegis.authorization.v1" must be within a directory "aegis/authorization/v1" relative to root but were in directory "proto/authorization".

(PACKAGE_DIRECTORY_MATCH)